org.fsf.pspp.yml

```diff
@@ -0,0 +1,33 @@
+app-id: org.fsf.pspp
```

Comment:
shouldn't it be `org.gnu.pspp` ?

Reply:
I thought that I'd found other flatpaks that use org.fsf, but I don't see any. I do see org.gnu.emacs. I guess it should be org.gnu.pspp.

Reply:
what does upstream think? (ie the developers if that project)

Reply:
For PSPP? I'm the lead developer. I don't have a strong opinion.

Reply:
ok so then we can discuss the app-id for `gtk_application_new()`. and that should be the one used here too.

Reply:
tbf `org.fsf` or `org.gnu` isn't different to me, but I'd consider the GNU project -> `org.gnu`.

Reply:
OK... I guess I'll come back when I can get an upstream release with all this updated then.